src/platform/packages/shared/serverless/settings/observability_project/index.ts

```diff
@@ -22,7 +22,7 @@ export const OBSERVABILITY_PROJECT_SETTINGS = [
   settings.OBSERVABILITY_APM_ENABLE_TABLE_SEARCH_BAR,
   settings.OBSERVABILITY_APM_ENABLE_SERVICE_INVENTORY_TABLE_SEARCH_BAR,
   settings.OBSERVABILITY_ENTITY_CENTRIC_EXPERIENCE,
-  settings.OBSERVABILITY_AI_ASSISTANT_SIMULATED_FUNCTION_CALLING,
-  settings.OBSERVABILITY_AI_ASSISTANT_SEARCH_CONNECTOR_INDEX_PATTERN,
+  // settings.OBSERVABILITY_AI_ASSISTANT_SIMULATED_FUNCTION_CALLING,
+  // settings.OBSERVABILITY_AI_ASSISTANT_SEARCH_CONNECTOR_INDEX_PATTERN,
```

Comment:
Should these be disabled only when in `essentials` tier?

Reply:
Sorry, this is WIP. The intention is that these two settings are not included for the essentials tier. The settings are included here:

https://github.com/elastic/kibana/blob/71006bd7bda530fe4758759492386161d214ad21/x-pack/solutions/observability/plugins/serverless_observability/server/plugin.ts#L29-L32

One approach would be something like:

```ts
pluginsSetup.serverless.setupProjectSettings(
  tier === "essentials"
    ? OBSERVABILITY_ESSENTIALS_PROJECT_SETTINGS
    : OBSERVABILITY_PROJECT_SETTINGS
);
```

or do you suggest I use `core.pricing.registerProductFeatures(...)` and use that directly (this is in the `setup` method)

Reply:
> or do you suggest I use core.pricing.registerProductFeatures(...) and use that directly

Yes please, that's the recommended approach we agreed with the core team to follow for easier maintenance and integration with other core services in the future :) 

So it would become something like:
```tsx
core.pricing.registerProductFeatures([
  {
    id: 'observability:complete-navigation',
    description: 'All the complete offer product features are reachable through the sidebar navigation.',
    products: [
      { name: 'observability', tier: 'complete' },
    ],
  },
])
```

Reply:
Thanks @rudolf ! Let me know if you create an issue on your end to tackle this so I can mark this as blocked, and refer to yours.